packages/web-forms/vite.config.ts

```diff
@@ -50,20 +57,36 @@ if (webkitFlakinessMitigations) {
 	globalSetup.push('./tests/globalSetup/mitigate-webkit-flakiness.ts');
 }
 
+const copyConfigFile = viteStaticCopy({
+	targets: [
+		{
+			src: 'src/demo/config.json',
```

Comment:
Why not just have this in assets and `import` it like other static files (e.g. `import demoConfig from 'whatever/demo/config.json?raw`)? Its inclusion in each build would be determined by the effective module graph, without need for an extra build plugin and out of band filesystem action.

Reply:
Doing that puts the content of the json file in `dist-demo/assets/index-<hash>.js`, which makes it difficult to update configuration value in the built artifact.

Reply:
Gotcha. Fine for now, I'll think about it for a followup!